CDCgov/prime-reportstream
# Running the smoke test sequentially

This PR updates the smoketest code so that the tests run sequentially when configured to

Test Steps:
1. Run the smoke tests locally to make sure they pass

I ran the CI build several times and it consistently passed

## Changes
- Updates the starting of each smoketest to that they run sequentially
- Fixes the UP test to verify the item lineage against the expected receivers

## Checklist

### Testing

### Process

## Linked Issues


## To Be Done
- #10289 

## Specific Security-related subjects a reviewer should pay specific attention to
